crypto/bio/bss_conn.c

```diff
@@ -333,7 +342,16 @@ static int conn_write(BIO *b, const char *in, int inl)
     }
 
     clear_socket_error();
-    ret = writesocket(b->num, in, inl);
+# ifndef OPENSSL_NO_KTLS
+    if (BIO_should_ktls_ctrl_msg_flag(b)) {
+        ret = ktls_send_ctrl_message(b->num, data->record_type, in, inl);
+        if (ret >= 0) {
+            ret = inl;
+            BIO_clear_ktls_ctrl_msg_flag(b);
+        }
+    } else
+# endif
+      ret = writesocket(b->num, in, inl);
```

Comment:
wrong indentation here - should be 2 spaces more.

Reply:
And now you've got conflicting style rules because if {} means else{}.  Perhaps best to duplicate the line of code and use an #else for the non-ktls case.

Reply:
I think it would be reasonable exception to ignore this rule. However the indentation is wrong anyway.

Reply:
Oops, thanks.  Fixed.